schema/snapcraft.yaml

```diff
@@ -88,6 +88,12 @@ properties:
           stop-command: ["daemon"]
         additionalProperties: false
         properties:
+          desktop:
+            type: string
+            description: path to a desktop file representing the app, relative to the prime directory
+          icon:
```

Comment:
This is here to allow overriding the icon in the upstream desktop file with a custom one for the snap.

For example, for webbrowser-app the Icon line for the input desktop file (before transformation) is:

    Icon=@CMAKE_INSTALL_FULL_DATADIR@/webbrowser-app/webbrowser-app.png

Typically on a deb system that would end up being:

    Icon=/usr/share/webbrowser-app/webbrowser-app.png

But when building a snap from the source tree, the line in the resulting desktop file in the prime directory will look like this:

    Icon=/share/webbrowser-app/webbrowser-app.png

Allowing to override the icon in snapcraft.yaml means upstream doesn’t need to care to support snaps.